runbotics/runbotics-orchestrator-ui/src/src-app/Actions/excel.actions.ts

```diff
@@ -281,6 +281,60 @@ const getExcelActions: () => Record<string, IBpmnAction> = () => ({
             },
         },
     },
+    'excel.insertColumnsBefore': {
+        id: ExcelAction.INSERT_COLUMNS_BEFORE,
+        label: translate('Process.Details.Modeler.Actions.Excel.InsertColumnsBefore.Label'),
+        script: ExcelAction.INSERT_COLUMNS_BEFORE,
+        runner: Runner.DESKTOP_SCRIPT,
+        system: ActionSystem.WINDOWS,
+        form: {
+            schema: {
+                type: 'object',
+                properties: {
+                    input: {
+                        title: translate('Process.Details.Modeler.Actions.Common.Input'),
+                        type: 'object',
+                        properties: {
+                            column: {
+                                title: translate('Process.Details.Modeler.Actions.Excel.StartColumn'),
+                                type: 'string',
+                            },
+                            amount: {
+                                title: translate('Process.Details.Modeler.Actions.Excel.ColumnsAmount'),
```

Comment:
if you want to use it in different actions (e.g. insert before/after) then:
- translate key should include info that's for INSERTION
OR
- translation should be "amount of columns" or sth like this not "amount of columns to INSERT"
-